evalml/automl/automl_search.py

```diff
@@ -857,16 +858,34 @@ def _handle_keyboard_interrupt(self):
             else:
                 leading_char = ""
 
-    def search(self, show_iteration_plot=True):
+    def search(self, show_iteration_plot=True, timing=None):
         """Find the best pipeline for the data set.
 
         Args:
             show_iteration_plot (boolean, True): Shows an iteration vs. score plot in Jupyter notebook.
                 Disabled by default in non-Jupyter enviroments.
+            timing (str, None): Shows timing of the batches and the individual timings of each pipeline.
+                Default: None
+                log=prints out batch/pipeline timing to console.
```

Comment:
Since the only options are "log" or None, I vote that we switch to a boolean flag for this, something like `log_timing` which defaults to false. It'd make both our lives as the implementers easier (checking a boolean instead of string equality, not having to validate the input) and make it clearer for users to boot.